src/pseudopeople/loader.py

```diff
@@ -8,21 +10,40 @@
 from pseudopeople.constants.metadata import DatasetNames
 from pseudopeople.exceptions import DataSourceError
 
+if TYPE_CHECKING:
+    from pseudopeople.utilities import DATAFRAME, ENGINE
+
+
+def load_standard_dataset_file(
+    data_path: Path, user_filters: List[Tuple], engine: ENGINE = "pandas"
+) -> DATAFRAME:
+    if engine == "pandas":
+        if data_path.suffix != ".parquet":
```

Comment:
Shouldn't these errors should be thrown whether the engine is pandas or modin?

Reply:
Yes, you're right. This first error was put inside the if because it only makes works if data_path is a single file and not a directory. I've made this more explicit.